pages/linux/systemctl.md

```diff
@@ -9,19 +9,19 @@
 
 - Start/Stop/Restart/Reload a service:
 
-`systemctl start/stop/restart/reload {{unit}}`
+`systemctl {{start|stop|restart|reload}} {{unit}}`
 
 - Show the status of a unit:
 
 `systemctl status {{unit}}`
 
 - Enable/Disable a unit to be started on bootup:
 
-`systemctl enable/disable {{unit}}`
+`systemctl {{enable|disable}} {{unit}}`
 
 - Mask/unmask a unit to prevent enablement and manual activation:
```

Comment:
Ooops this should be reverted too @navarroaxel (`unmask` → `Unmask`).  We did that in another PR but...

Reply:
Done https://github.com/tldr-pages/tldr/pull/5281